internal/pkg/skuba/upgrade/node/versions.go

```diff
@@ -99,6 +102,10 @@ func (nviu NodeVersionInfoUpdate) NodeUpgradeableCheck(client clientset.Interfac
 	if err != nil {
 		return err
 	}
+	//Check if node OS SP compatible with kubernetes.latestVersion
```

Comment:
Are you sure that the node information is up to date? I would refresh the cache by checking the target OS (call of ` 	target.IsSUSEOS()` before running this function. (Like you did in the previous PR).